tokens.xml

```diff
@@ -9837,6 +10037,60 @@ Whenever a card is put into your graveyard from anywhere, you may return it to y
         </card>
+        <card>
+            <name>Day</name>
+            <prop>
+                <type>State</type>
+                <maintype>State</maintype>
+            </prop>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/9/c/9c0f7843-4cbb-4d0f-8887-ec823a9238da.jpg?1630606483">MID</set>
+            <reverse-related exclude="exclude">Arlinn, the Pack's Hope</reverse-related>
+            <reverse-related exclude="exclude">Baneblade Scoundrel</reverse-related>
+            <reverse-related exclude="exclude">Bird Admirer</reverse-related>
+            <reverse-related exclude="exclude">Brimstone Vandal</reverse-related>
+            <reverse-related exclude="exclude">Brutal Cathar</reverse-related>
+            <reverse-related exclude="exclude">Burly Breaker</reverse-related>
+            <reverse-related exclude="exclude">Celestus Sanctifier</reverse-related>
+            <reverse-related exclude="exclude">Component Collector</reverse-related>
+            <reverse-related exclude="exclude">Curse of Leeches</reverse-related>
+            <reverse-related exclude="exclude">Fangblade Brigand</reverse-related>
+            <reverse-related exclude="exclude">Firmament Sage</reverse-related>
+            <reverse-related exclude="exclude">Gavony Dawnguard</reverse-related>
+            <reverse-related exclude="exclude">Graveyard Trespasser</reverse-related>
+            <reverse-related exclude="exclude">Harvesttide Infiltrator</reverse-related>
+            <reverse-related exclude="exclude">Hound Tamer</reverse-related>
+            <reverse-related exclude="exclude">Kessig Naturalist</reverse-related>
+            <reverse-related>Night</reverse-related>
+            <reverse-related exclude="exclude">Obsessed Astronomer</reverse-related>
+            <reverse-related exclude="exclude">Outland Liberator</reverse-related>
+            <reverse-related exclude="exclude">Reckless Stormseeker</reverse-related>
+            <reverse-related exclude="exclude">Shady Traveler</reverse-related>
+            <reverse-related exclude="exclude">Spellrune Painter</reverse-related>
+            <reverse-related exclude="exclude">Sunrise Cavalier</reverse-related>
+            <reverse-related exclude="exclude">Sunstreak Phoenix</reverse-related>
+            <reverse-related exclude="exclude">Suspicious Stowaway</reverse-related>
+            <reverse-related exclude="exclude">Tavern Ruffian</reverse-related>
+            <reverse-related exclude="exclude">The Celestus</reverse-related>
+            <reverse-related exclude="exclude">Tireless Hauler</reverse-related>
+            <reverse-related exclude="exclude">Tovolar, Dire Overlord</reverse-related>
+            <reverse-related exclude="exclude">Tovolar's Huntmaster</reverse-related>
+            <reverse-related exclude="exclude">Vadrik, Astral Archmage</reverse-related>
+            <reverse-related exclude="exclude">Village Watch</reverse-related>
```

Comment:
Hmm, why did you put the exclude tag for most of those Daybound cards?

<br>

There are also png pictures at scryfall: https://c1.scryfall.com/file/scryfall-cards/png/front/9/c/9c0f7843-4cbb-4d0f-8887-ec823a9238da.png?1630606483

I did not test it but I assume the jpg looks not as great with its white corners.
Wizards images looks to be png, scryfall mostly jpg in the rest of the file...

Reply:
they all have white corners, cockatrice intentionally adds a black border to all cards to sort of counteract this

Reply:
the exclude tag means they don't get created together with normal tokens, this is what we want for states

Reply:
Everything being reverse-related is basically just because of me copy pasting the same format for all of them. Some of them should definitely be excluded because they make actual tokens besides Day.

Reply:
>they all have white corners, cockatrice intentionally adds a black border to all cards to sort of counteract this

Thanks for clarifying 👍 

>the exclude tag means they don't get created together with normal tokens, this is what we want for states

Sure, but none of those do create other tokens... hence the question.
There is mostly only Arlinn and Tovolar's Huntmaster doing so: https://scryfall.com/search?q=e%3Amid+o%3Adaybound&unique=cards&as=grid&order=name